src/sentry/api/endpoints/organization_details.py

```diff
@@ -180,6 +182,11 @@ def _has_sso_enabled(self):
         org = self.context["organization"]
         return AuthProvider.objects.filter(organization_id=org.id).exists()
 
+    @property
+    def has_api_auth_provider(self):
+        org = self.context["organization"]
+        return features.has("organizations:api-auth-provider", org)
```

Comment:
Kept the org variable in this function to keep code consistent with similar functions above